### ISSUE TITLE
Bad git executable for gitinspectorgui on Windows 10

### ISSUE THREAD

**Issue by davbeek** [davbeek/gitinspectorgui#13]
![image](https://github.com/user-attachments/assets/b204cbbe-6c96-4975-82b2-6bdb44247bbf)

On a Windows 10 system from a student, this error was reported when opening the installed gitinspectorgui via the Windows start menu.

**Comment by davbeek** [comment]
We should test giginspectorgui on a windows 10 system via a user who has not installed any of our python tools to see if it works there.

**Comment by Alberth289346** [comment]
From the error message, it seems that git was not installed at the CLI. Maybe they only installed an app?

You could check for existence of it during install or startup.

**Comment by davbeek** [comment]
I was able to replicate the issue on Windows by uninstalling "Git for Windows" which I had installed a long time ago for all users. Opening GitinspectorGUI then indeed resulted in the "Bad git executable" error.

In principle, we could include Git with our GitinspectorGUI app. The problem with that, apart from complexity, is that the minimal Git portable version needed for Windows is around 300MB, whereas our gitinspectorgui-setup.exe is only 17MB. Therefore, I think that for Windows, we should add instructions on how to install "Git for Windows" which is required to run gitinspectorgui on Windows. 

For macOS, we need to find out if all Macs come with a usable version of git and if not, see how we deal with that.

We don't officially support Linux yet, so we can forget about that for now.
